### DISCUSSION TITLE
problems with input type=file in a repeatable field

### DISCUSSION THREAD

**Discussion by linpi1** [joomengine/Joomla-Component-Builder#845] [Extention Issues]
I use a field type=file in a repeatable field. When I select a file to upload, in the json string of the repeatable field, in the "upload file field", there is only:
`C:\\fakepath\\file-name`
After saving, the path `C:\\fakepath\\` block the admin view as well, therefore I removed this (windows) C:\fakepath.
That means there is no file in /tmp as usual, to be able to move it to the destination. 
Has anyone an idea to overcome this problem be able to upload a file in a repeatable field ?

<img width="629" alt="Bildschirmfoto 2021-12-28 um 17 26 15" src="https://user-images.githubusercontent.com/20855980/147586667-3f9c895e-beda-4405-81f6-4e5308e9f82e.png">

**Comment by Llewellynvdm** [comment]
Using the file field type in a repeatable field will never work out the box, JCB does not do anything to make those fields work, you need to do the work. Watch [this tutorial](https://www.youtube.com/watch?v=o482sK4DxkM&list=PLQRGFI8XZ_wtGvPQZWBfDzzlERLQgpMRE&index=57) of the normal "not repeatable field" implementation, and then adjust for your use-case. There is also a [drag and drop](https://www.youtube.com/watch?v=UvzDyVQyHDI&list=PLQRGFI8XZ_wtGvPQZWBfDzzlERLQgpMRE&index=58) option which will work better for multiple files.

**Reply by linpi1** [reply]
;-) funny enough,  I also discovered you tutorial  "How to use the file field type to upload a file in JCB" this morning .... I will look at it next year.
